tests/fast-integration/monitoring-test/monitoring-test.js

```diff
@@ -0,0 +1,92 @@
+const uuid = require("uuid");
+
+const {
+  assert
+} = require("chai");
+
+const {
+  waitForPodWithLabel,
+} = require("../utils");
+
+const {
+  prometheusPortForward,
+  getPrometheusActiveTargets,
+  getPrometheusAlerts,
+  getPrometheusRuleGroups,
+} = require("../monitoring/client")
+
+const {
+  shouldIgnoreTarget,
+  shouldIgnoreAlert,
+  buildScrapePoolSet,
+  assertTimeSeriesExist,
+} = require("../monitoring/helpers");
+
+describe("Monitoring test", function () {
+  this.timeout(30 * 60 * 1000); // 30 min
+  this.slow(5 * 1000);
+
+  var cancelPortForward;
+
+  before(() => {
+    cancelPortForward = prometheusPortForward();
+  })
+
+  after(() => {
+    cancelPortForward()
+  })
+
+  it("All Prometheus targets should be healthy", async () => {
+    let activeTargets = await getPrometheusActiveTargets();
+    let unhealthyTargets = activeTargets
+      .filter(t => !shouldIgnoreTarget(t) && t.health != "up")
+      .map(t => t.discoveredLabels.job);
```

Comment:
can you add the other labels and the error message in the logs? and not only the job label.
We had before requests from other teams to include more information when the test for targets is failing, so that they more easily figure out the root cause for the failure.

Reply:
Done